libs/datamodel/connectors/datamodel-connector/src/lib.rs

```diff
@@ -221,3 +221,64 @@ impl ConnectorCapabilities {
         self.capabilities.contains(&capability)
     }
 }
+
+impl std::fmt::Display for ConnectorCapability {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let name = match self {
+            ConnectorCapability::ScalarLists => "ScalarLists",
+            ConnectorCapability::RelationsOverNonUniqueCriteria => "RelationsOverNonUniqueCriteria",
+            ConnectorCapability::MultipleIndexesWithSameName => "MultipleIndexesWithSameName",
+            ConnectorCapability::Enums => "Enums",
+            ConnectorCapability::Json => "Json",
+            ConnectorCapability::AutoIncrement => "AutoIncrement",
+            ConnectorCapability::AutoIncrementAllowedOnNonId => "AutoIncrementAllowedOnNonId",
+            ConnectorCapability::AutoIncrementMultipleAllowed => "AutoIncrementMultipleAllowed",
+            ConnectorCapability::AutoIncrementNonIndexedAllowed => "AutoIncrementNonIndexedAllowed",
+            ConnectorCapability::RelationFieldsInArbitraryOrder => "RelationFieldsInArbitraryOrder",
+            ConnectorCapability::InsensitiveFilters => "InsensitiveFilters",
+            ConnectorCapability::CreateMany => "CreateMany",
+            ConnectorCapability::CreateManyWriteableAutoIncId => "CreateManyWriteableAutoIncId",
+            ConnectorCapability::WritableAutoincField => "WritableAutoincField",
+            ConnectorCapability::CreateSkipDuplicates => "CreateSkipDuplicates",
+            ConnectorCapability::UpdateableId => "UpdateableId",
+            ConnectorCapability::JsonFiltering => "JsonFiltering",
+            ConnectorCapability::JsonFilteringJsonPath => "JsonFilteringJsonPath",
+            ConnectorCapability::JsonFilteringArrayPath => "JsonFilteringArrayPath",
+            ConnectorCapability::CompoundIds => "CompoundIds",
+            ConnectorCapability::ForeignKeys => "ForeignKeys",
+        };
+
+        write!(f, "{}", name)
+    }
+}
```

Comment:
This is to serialize the `ConnectorCapability` enum without having to do JSON serialization. Tom was a bit reluctant to it

Reply:
Not really a fan, you can't you reuse the parse pair mechanism here again?

Reply:
Fixed by macro as discussed 👍